homeassistant/components/climate/ecobee.py

```diff
@@ -348,6 +348,21 @@ def set_auto_temp_hold(self, heat_temp, cool_temp):
 
         self.update_without_throttle = True
 
+    def set_fan_mode(self, fan_mode):
+        """Set the fan mode.  Valid values are "on" or "auto"."""
+        if (fan_mode.lower() != STATE_ON) and (fan_mode.lower() != STATE_AUTO):
+            error = "Invalid fan_mode value:  Valid values are 'on' or 'auto'"
+            _LOGGER.error(error)
+            return
+
+        cool_temp = self.thermostat['runtime']['desiredCool'] / 10.0
```

Comment:
In general, it's usually a good idea to put these strings as constants.

It's also unclear from this code why we're multiplying by 10.0. I see some multiplication in the pyecboee code-too.

Maybe it's worth updating `set_fan_mode` to make `cool_temp` and `heat_temp` optional? Even if the API needs it, pyecobee should have access to pass that in if it needs it.

Reply:
The multiplier is because ecobee accepts 3 digit temp ranges. So 68 degrees becomes 680 degrees.

As far as the temp goes, I just kept what the existing functionality had.